app/src/main/java/com/bottlerocketstudios/android_maps_demo/ui/theme/Color.kt

```diff
@@ -0,0 +1,8 @@
+package com.bottlerocketstudios.android_maps_demo.ui.theme
```

Comment:
Refactor all files under `com.bottlerocketstudios.android_maps_demo` to `com.bottlerocketstudios.mapsdemo`